modules/raft-api/src/test/java/org/apache/ignite/internal/raft/RaftGroupConfigurationSerializerTest.java

```diff
@@ -61,10 +65,12 @@ void serializationAndDeserializationWithNulls() {
 
     @Test
     void v1CanBeDeserialized() {
```

Comment:
If this PR is not going to be included in the 3.0 release, then changes to this test need to be reverted and another test needs to be added (`v2CanBeDeserialized()`)

Reply:
added